api/resources/ceph/v1/resources.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package api.resources.ceph.v1;
+
+option go_package = "github.com/koor-tech/data-control-center/gen/go/api/resources/ceph;ceph";
+
+message CustomResource {
```

Comment:
Let's make this more generic instead of focusing it on custom resources.
In the end what this encapsulates is a K8S "object"/resouce type, name, namespace and YAML content.
The YAML content would still be the whole object's YAML, but adding these fields for name, namespace, to make it easier to work with on front- and backend-side.
What do you think?